### PR TITLE
Add otel tracer

### PR DESCRIPTION
**Pull request by afzalbin64**

### CODE REVIEW

File: examples/hotrod/services/driver/redis.go [hunk 1]
```diff
@@ -22,20 +22,21 @@ import (
...
 
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+
 	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/ext"
 	"go.uber.org/zap"
 
 	"github.com/jaegertracing/jaeger/examples/hotrod/pkg/delay"
 	"github.com/jaegertracing/jaeger/examples/hotrod/pkg/log"
-	"github.com/jaegertracing/jaeger/examples/hotrod/pkg/tracing"
```
Comment (albertteoh) [inline examples/hotrod/services/driver/redis.go:31]: Removing this is causing the errors below. I think you need to keep it.

Comment (afzalbin64) [reply examples/hotrod/services/driver/redis.go:31]: Yes, but `tracing.Init()` doesn't return opentelemetry instance as defined in the struct.

Comment (afzalbin64) [reply examples/hotrod/services/driver/redis.go:31]: In your PR you mentioned `tracing.InitOTEL()` to  adjust the function call.

Comment (albertteoh) [reply examples/hotrod/services/driver/redis.go:31]: Ah yup, you're right; sorry, wasn't sure what you were trying to achieve.

Comment (afzalbin64) [reply examples/hotrod/services/driver/redis.go:31]: added that missing tracing.InitOTEL() func.

**Comment by afzal442** [comment]
Done!

**Pull request closed without merging**